ramls/request.json

```diff
@@ -225,10 +230,10 @@
   },
   "additionalProperties": false,
   "required": [
-    "itemId",
```

Comment:
- Revert this change
- add holdingsRecordId (required)

Reply:
Resolved